src/OpenTelemetry.Contrib.Instrumentation.AWS/Implementation/IRequestContextAdapter.cs

```diff
@@ -0,0 +1,25 @@
+// <copyright file="IRequestContextAdapter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Collections.Generic;
+
+namespace OpenTelemetry.Contrib.Instrumentation.AWS.Implementation;
+internal interface IRequestContextAdapter
```

Comment:
Not sure what the guidelines in this project are, but if there is no guideline against it, I would put this interface and the two implementations into a single .cs file.

Reply:
In this case I would prefer to keep definition per file. The interface implementations are not that small. Also looking at the project sub-folder you may guess just by names how many implementations the interface has.

Reply:
This interface has been removed during code simplifications.